### ISSUE TITLE
Feature Request: Add lighting effect layers

### ISSUE THREAD

**Issue by CodeGuro** [polychromatic/polychromatic#176]
This is a nice piece of software you folks made. Once I've had it set up, it was pretty robust. The only issue I had was having to disable the UEFI secure boot to get it to work (which is a little odd, but no biggie for me).

Anyway, as a migrating Windows -> Linux user, one of the features I miss from the Razer Synapse v3 effect layers. Polychromatic lacks this feature - only one effect can be selected and applied at any one time. It would be nice if you could stack effects on top of one another; for example Reactive + Static effects, Or Wave + Spectrum. Or any combination of any of the other effects. Preferably with some kind of configurable priority setting, for a customizeable experience.

**Comment by lah7** [comment]
Thanks! Be assured the software is still being refined, so more improvements are on the list!

The feature you've requested is a little complex at the moment, but it's something I'd like to build in the future eventually. Currently, apart from ripple, the effects are actually powered by the hardware itself.

**Comment by ricknetter** [comment]
Was really hoping Reactive + Static was a possibility when I first saw screenshots (it showed checkboxes, instead of a dropdown I have now). Now I see it's tricky - hope it can be resolved!

**Comment by koyuawsmbrtn** [comment]
Yep, please make that happen or I have to go back to Windows :disappointed:

**Comment by TimSousa1** [comment]
3 years have passed since this thread was opened. Is this feature out already? I'm not quite sure how to use it

**Comment by lah7** [comment]
Sorry, development has been slow 🐢  but it's [on the roadmap](https://docs.polychromatic.app/roadmap/). 

Some work is needed under the hood, then the application will be in a much better state. I hope it'll be worth the wait, although it's not helping I haven't worked on this project for many of the months in 2022!

For now, there's the effects that come from OpenRazer, and then an [effect editor](https://docs.polychromatic.app/controller/effects/) if your device supports it - sort of like painting over the keys. There is no layering or custom software effects yet.

**Comment by koyuawsmbrtn** [comment]
Since we're onto it it would be nice to see the ability to import Windows configuration files into Polychromatic, of course in a seperate ticket
